bin/coding-game-service.js.in

```diff
@@ -55,9 +57,11 @@ const Service = imports.lib.service;
 function loadTimelineDescriptors(cmdlineFile) {
     let filesToTry = [
         cmdlineFile,
-        Gio.File.new_for_path(GLib.build_filenamev([GLib.get_user_config_dir(),
-                                                    'com.endlessm.CodingGameService',
-                                                    'timeline.json'])),
+        Gio.File.new_for_path(GLib.build_filenamev([
```

Comment:
I don't think its that bad - hanging brackets is how things are done for lists generally